flash-lso/src/amf3/read.rs

```diff
@@ -151,12 +151,17 @@ fn parse_element_int(i: &[u8]) -> AMFResult<'_, Rc<Value>> {
 pub struct AMF3Decoder {
     /// The table used to cache repeated byte strings
     pub string_reference_table: Vec<Vec<u8>>,
+
     /// The table used to cache repeated trait definitions
     pub trait_reference_table: Vec<ClassDefinition>,
+
     /// The table used to cache repeated objects
     pub object_reference_table: Vec<Rc<Value>>,
+
     /// Encoders used for handling externalized types
     pub external_decoders: HashMap<String, ExternalDecoderFn>,
+
+    object_id: i64,
```

Comment:
can you please document this field for the future? (including why it's not `ObjectId`)
Also, any reason why it differs in signed-ness from ObjectId?

Reply:
Added docs. 
It doesn't differ from ObjectId, they are both `i64`s?

Reply:
My bad, could have sworn I saw `u64` somewhere :/